.github/workflows/coder.yaml

```diff
@@ -338,14 +338,18 @@ jobs:
           else
             echo ::set-output name=cover::false
           fi
-          gotestsum --junitfile="gotests.xml" --jsonfile="gotestsum.json" --packages="./..." --debug -- -parallel=8 -timeout=5m -short -failfast $COVERAGE_FLAGS
+          set +e
+          gotestsum --junitfile="gotests.xml" --jsonfile="gotestsum.json" --packages="./..." --debug -- -parallel=8 -timeout=30s -short -failfast $COVERAGE_FLAGS
           ret=$?
           if ((ret)); then
             # Eternalize test timeout logs because "re-run failed" erases
             # artifacts and gotestsum doesn't always capture it:
             # https://github.com/gotestyourself/gotestsum/issues/292
-            echo "Checking gotestsum.json for panic trace:"
-            grep -A 999999 'panic: test timed out' gotestsum.json
```

Comment:
This was pretty loud (so many logs after timeout), so I decided to only capture the stack trace, and while I was at it, only when the criteria is met.